lms/templates/edraak_university/university_id.html

```diff
@@ -0,0 +1,58 @@
+## mako
+
+<%inherit file="/main.html" />
+<%namespace name='static' file='/static_content.html'/>
+<%!
+    from django.utils.translation import ugettext as _
+    from django.core.urlresolvers import reverse
+    from django.conf import settings
+%>
+<%block name="bodyclass">view-in-course view-university-id</%block>
+
+<%block name="headextra">
+    <%static:css group='style-course-vendor'/>
+    <%static:css group='style-course'/>
+</%block>
+
+
+<%block name="pagetitle">${_("{course_number} University ID").format(course_number=course.display_number_with_default) | h}</%block>
+<%block name="nav_skip">#course-university-id</%block>
+
+<%include file="/courseware/course_navigation.html" args="active_page='university_id'" />
+
+<div class="container">
+    <div class="profile-wrapper">
+        <div class="university-id" id="course-university-id" aria-label="${_('University ID')}">
+            <form class="university-id-form" method="post">
+                <h1>${_('Edraak University ID')}</h1>
+                <div class="form-help">
+                    % if has_valid_information:
+                        <p>
+                            ${_('The information bellow have been successfully submitted '
+                                'to your university coordinator.')}
+                        </p>
+
+                        <p>
+                            ${_('At anytime you can review and amend the information.')}
+                        </p>
+                    % else:
+                        <p>
+                            ${_('Please enter your the following information to be used by the university coordinator.')}
```

Comment:
Change to:
Your following University information is needed for your grade reports. Please enter  the required information, and verify it is correct.
edX red**